extensions/opentelemetry/opentelemetry/runtime/pom.xml

```diff
@@ -27,6 +27,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This will add a HTTP server to all applications using OpenTelemetry, correct?

Is that desired for Reactive Messaging applications?

Reply:
This was to obtain the Vert.x Routing Context, but I may be able to rewrite this without a direct dependency.